core/ftn_address.js

```diff
@@ -135,7 +135,7 @@ module.exports = class Address {
     static fromString(addrStr) {
         const m = FTN_ADDRESS_REGEXP.exec(addrStr);
 
-        if (m) {
+        if (m && m[2] && m[3]) {
```

Comment:
I think we need to revert this part, right?

Reply:
The test for m[2] _might_ not be needed.

To be honest, I never found a way to cause m[2] to be null.  I just figured if testing for m[3] was necessary, testing for m[2] was probably not a bad idea.  Perhaps I was being a bit too defensive.

Anyways... I tried removing the test for m[3] just now, and found that entering a numeric value in the address field will cause an error and disconnect me.  So I believe that test is still needed.

I'm curious to know if the tests for m[2] and m[3] might become unnecessary if FTN_ADDRESS_REGEXP was tweaked a bit.  Just a thought.